lib/is-dir.js

```diff
@@ -0,0 +1,18 @@
+var fs = require('fs')
+
+function isDirectory (path, callback) {
+  fs.stat(path, function (err, stats) {
+    if (err) return callback(err)
+
+    callback(null, stats.isDirectory())
+  })
+}
+
+function isDirectorySync (path) {
+  return fs.statSync(path).isDirectory()
```

Comment:
Isn't `isDirectorySync` supposed not to throw if the path doesn't exists? Because it will throw if you don't catch `statSync` for `ENOENT` errors.

The strange thing is that `isDirectory` swallows the errors and `isDirectorySync` not.. they don't have the same behavior.

@overlookmotel I think you have the exact same problem here: https://github.com/overlookmotel/fs-extra-promise

Reply:
@yvele I don't see it as a bug. I think it *should* throw if the path doesn't exist, same as calling `fs.stat()`/`fs.statSync()` would.